packages/pwa-kit-create-app/assets/templates/@salesforce/retail-react-app/config/default.js.hbs

```diff
@@ -8,6 +8,31 @@
 const sites = require('./sites.js')
 module.exports = {
     app: {
+        // Commerce shopping agent configuration for embedded messaging service
+        // This enables an agentic shopping experience in the application
+        // This property is set to a serialized JSON string that contains the configuration for the commerce agent.
+        // The JSON string is set to the value of the COMMERCE_AGENT_SETTINGS environment variable.
+        // If the COMMERCE_AGENT_SETTINGS environment variable is not set, the feature is disabled.
+        commerceAgent: process.env.COMMERCE_AGENT_SETTINGS || JSON.stringify({
```

Comment:
Thanks for adding and commenting this part of the config. 

Sorry my bad, there's actually another similar file to update as well: in 'assets/bootstrap/js/config/default.js.hbs'.

Depending on what a user chooses while generating a project:
- If they say yes to using template extensibility, the generator will use the 'assets/bootstrap/' folder
- Otherwise, generator will use the 'assets/templates/@salesforce/retail-react-app' folder

In general, if you search for a filename and a similar filename also exists in the pwa-kit-create-app package, then there's a good chance that you'll need to update them too.